clients/mtaclient/operations/get_mta_operations_parameters.go

```diff
@@ -117,6 +119,17 @@ func (o *GetMtaOperationsParams) SetLast(last *int64) {
 	o.Last = last
 }
 
+// WithMtaID adds the mtaID to the get operations params
+func (o *GetMtaOperationsParams) WithMtaID(mtaID *string) *GetMtaOperationsParams {
```

Comment:
Again same question, have you generated this?

Reply:
The model generation is done by go-swagger: https://github.com/go-swagger/go-swagger